tests/test_check_store_wiring.py

```
@@ -0,0 +1,415 @@
+            "    MIGRATIONS = []\n"
+            "\n"
+            "\n"
+            "class NewStore(BaseStore):\n"
+            "    SCHEMA = 'CREATE TABLE IF NOT EXISTS new_store (id INTEGER PRIMARY KEY);'\n"
+            "    MIGRATIONS = []\n",
+            "feat: add NewStore alongside MetricsStore",
+        )
+        _checkout(repo, "main")
+        _git(repo, "merge", "pr", "--no-edit")
+
+        violations, waived = csw.check_store_wiring(base_tip, repo)
```

Ruff (0.16.1)

[warning] Unpacked variable `waived` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)